tekton/ci-lint.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: golangci-lint
+spec:
+  inputs:
+    params:
+    - name: package
+      description: package (and its children) under test
+      default: github.com/tektoncd/pipeline
+    - name: flags
+      description: flags to use for the test command
```

Comment:
i think this description might not be quite right? ("for the lint command"?)

Reply:
:man_facepalming: